app/views/admin/users/_form.html.erb

```diff
@@ -1,35 +1,47 @@
-<%= admin_form_for([:admin, @user.becomes(User)]) do |f| %>
-  <%= f.inputs "Details" do %>
-    <% if @user.new_record? %>
-      <% Pageflow::AccountPolicy::Scope
-           .new(current_user, Pageflow::Account).member_addable.each do |account| %>
-        <%= quota_state_description(:users, account) %>
+<% if @user.new_record? &&
+      Pageflow.config.quotas.get(:users, @user.initial_account).exhausted? %>
+  <%= render 'admin/users/quota_exhausted' %>
```

Comment:
* Let's pass an `account` local to the partial. This will make implementing the partial itself much easier.
* Let's rename the partial to `pageflow/admin/users/quota_exhausted`. The way we use it, it is API surface. So I think staying in our own namespace makes sense.
* Given that the partial is API surface, let's make sure it cannot get lost again and add a controller spec. Maybe we can use something like [stub_template](https://github.com/codevise/pageflow/blob/0f417b64d04c447cc5f0045ef7d695a4a0a62e32/spec/support/helpers/stub_template_in_helper_specs.rb) to actually make sure the correct locals are passed. Otherwise we should at least be able to use [the render_template matcher](https://relishapp.com/rspec/rspec-rails/docs/matchers/render-template-matcher)